src/css/override_components.scss

```diff
@@ -185,8 +185,8 @@ input:focus {
 
   [type='submit'] {
     background-color: #ffffff;
-    background-image: url('#{$static-img-path}/search.png');
```

Comment:
i don't think we need `search.png` or `search.svg` anymore now right? i think we can delete them now.